src/rgw/rgw_main.cc

```diff
@@ -327,15 +327,10 @@ int radosgw_Main(int argc, const char **argv)
 #if defined(WITH_RADOSGW_FCGI_FRONTEND)
   FCGX_Init();
 #endif
-
```

Comment:
this looks very suspicious--is this reversing part of zipper?

Reply:
unsure, is it possible to expand please which part of zipper the reversing is related to?

Reply:
I think the change I was looking at is no longer present